config/initializers/health_checks.rb

```diff
@@ -0,0 +1,41 @@
+class DelayedJobNeverRanCheck < OkComputer::Check
+  attr_reader :threshold
+
+  def initialize(threshold)
+    @threshold = threshold
+  end
+
+  def check
+    never_ran = Delayed::Job.where('run_at <= ?', 5.minutes.ago).count
```

Comment:
We might want to make that time dependent on a multiple of `Delayed::Worker.sleep_delay`